pkg/operator/ceph/file/controller.go

```diff
@@ -101,6 +101,18 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Build Handler function to return the list of filesystems
+	handerFunc, err := opcontroller.ObjectToCRMapper(mgr.GetClient(), &cephv1.CephFilesystemList{}, mgr.GetScheme())
```

Comment:
What about a helper method such as `StartWatchCluster()` that would call both `ObjectToCRMapper()` and `c.Watch()`? Just a small simplification for the child controllers.